easybuild/easyconfigs/b/binutils/binutils-2.25-GCC-4.9.3-binutils-2.25.eb

```diff
@@ -6,7 +6,7 @@ version = '2.25'
 homepage = 'http://directory.fsf.org/project/binutils/'
 description = "binutils: GNU binary utilities"
 
-toolchain = {'name': 'GCC', 'version': '4.9.3-binutils-%(version)s'}
+toolchain = {'name': 'GCC', 'version': '4.9.3-binutils-%s' % version}
```

Comment:
Actually I think this should stay, this works regardless of MNS, the other only works with the flat scheme, why leave something broken inside while we wait for a framework  fix?